submissions/alstep07/task-js-dom/app.js

```diff
@@ -0,0 +1,333 @@
+         'rating': '8.4',
+         'director': 'Makoto Shinkai',
+         'description': 'Two strangers find themselves linked in a bizarre way. When a connection forms, will distance be the only thing to keep them apart?'
+        },
+        {
+         'title': 'Eternal Sunshine of the Spotless Mind',
+         'year': '2004',
+         'rating': '8.3',
+         'director': 'Michel Gondry',
+         'description': 'When their relationship turns sour, a couple undergoes a medical procedure to have each other erased from their memories.'
+        }
+    ],
+    'horror': [    
+        {
+         'title': 'Psycho',
+         'year': '1960',
+         'rating': '8.5',
+         'director': 'Alfred Hitchcock',
+         'description': 'A Phoenix secretary embezzles $40,000 from her employer\'s client, goes on the run, and checks into a remote motel run by a young man under the domination of his mother.'
+        },
+        {
+         'title': 'The Shining',
+         'year': '1980',
+         'rating': '8.4',
+         'director': 'Stanley Kubrick',
+         'description': 'A family heads to an isolated hotel for the winter where a sinister presence influences the father into violence, while his psychic son sees horrific forebodings from both past and future.'
+        },
+        {
+         'title': 'Alien',
+         'year': '1979',
+         'rating': '8.4',
+         'director': 'Ridley Scott',
+         'description': 'After a space merchant vessel receives an unknown transmission as a distress call, one of the crew is attacked by a mysterious life form and they soon realize that its life cycle has merely begun.'
+        },
+        {
+         'title': 'Tumbbad',
+         'year': '2018',
+         'rating': '8.3',
+         'director': 'Rahi Anil Barve',
+         'description': 'A mythological story about a goddess who created the entire universe. The plot revolves around the consequences when humans build a temple for her first-born.'
+        },
+        {
+         'title': 'The Blue Elephant',
+         'year': '2014',
+         'rating': '8.1',
+         'director': 'Marwan Hamed',
+         'description': 'The story of Dr. Yehia, a psychotherapist at Al Abbasia hospital. He works in the department treating the criminally insane, only to find his best friend to be one of the patients. Trying to help his friend he finds himself going down the rabbit hole.'
+        }
+    ]
+};
+
+
+const main = document.querySelector('.main__container');
+const navBtn = document.querySelectorAll('.navbar__item');
+const genres = ['action', 'drama', 'comedy', 'western', 'romance', 'horror'];
+
+render('all');
+
+navBtn.forEach(function (btn,i){
+    btn.addEventListener('click', function(){
+        render(genres[i]);
+    })
+});
+
+function render(genre){
+
+    if (main.hasChildNodes()){
+        main.firstChild.remove();
+    }
+
+    let filmCards = document.createElement('div');
+    filmCards.className="main__genre";
```

Comment:
Use [`classList`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) it is more predictable then `className`.